_posts/2016-01-01-Markdown Reference.md

```diff
@@ -242,11 +251,11 @@ Then, anywhere in the document, you define your link label on a line by itself l
 [id]: http://example.com/  "Optional Title Here"
 ```
 
-In Typora, they will be rendered like so:
+<!-- In Typora, they will be rendered like so:
```

Comment:
why wrap them in `<!-- -->` comment?

Reply:
Oops. I thought it was duplication, but I see it wasn't. Fixed.